### PR TITLE
updated Playground User Manual

### PR DESCRIPTION
**Pull request by magua-io**
- Copied content written by Edward from [here](https://github.microstrategy.com/elin/embedding-sdk-docs/blob/playground-docs/content/pages/introduction-to-the-embedding-sdk/product-documentation.md).
- Added more sections and gifs, and remade all gifs.

Note: You can preview the README at https://github.com/MicroStrategy/playground/tree/add-playground-user-manual

Any feedbacks are welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,96 @@
-# playground
-embedding playground
+# Playground User Manual
```
Comment (xdumstr) [inline README.md:1]:
```suggestion
# Embedding SDK Playground User Manual
```

**Review by xdumstr — COMMENTED** [review]
Suggestions:

* Add a link to playground application. 
* Also add my Tutorial playlist link: https://www.youtube.com/playlist?list=PL2K8Xb9vcI4XHedNmio1JdWs3gI8kwfwR

**Review by elinmstr — APPROVED** [review]
Looks good to me for a start.

**Comment by magua-io** [comment]
> Suggestions:
> 
> * Add a link to playground application.
> * Also add my Tutorial playlist link: https://www.youtube.com/playlist?list=PL2K8Xb9vcI4XHedNmio1JdWs3gI8kwfwR

@xdumstr Changes have been made in the latest commit. Thanks for the feedback.

**Review by x-du — APPROVED** [review]

**Review by xdumstr — APPROVED** [review]
Thank you for the update! @magua-io

**Pull request merged**